airflow/www/static/js/grid/details/content/taskInstance/Logs/index.jsx

```diff
@@ -65,6 +70,9 @@ const Logs = ({
   const [selectedAttempt, setSelectedAttempt] = useState(1);
   const [shouldRequestFullContent, setShouldRequestFullContent] = useState(false);
   const [wrap, setWrap] = useState(false);
+  const [logLevelFilter, setLogLevelFilter] = useState('');
+  const [logGroupFilter, setLogGroupFilter] = useState('');
```

Comment:
Why did you go with calling it a log group? I'm trying to figure out if that is the best name or if we something like "file source"?

Reply:
Yeah, let's go with that